var/spack/repos/builtin/packages/intel-mpi/package.py

```diff
@@ -75,15 +83,31 @@ def setup_dependent_environment(self, spack_env, run_env, dependent_spec):
         spack_env.set('I_MPI_FC', spack_fc)
 
     def setup_dependent_package(self, module, dep_spec):
-        # Check for presence of bin64 or bin directory
-        if os.path.isdir(self.prefix.bin):
-            bindir = self.prefix.bin
-        elif os.path.isdir(self.prefix.bin64):
-            bindir = self.prefix.bin64
-        else:
-            raise RuntimeError('No suitable bindir found')
+        bindir = self.prefix.compilers_and_libraries.linux.mpi.intel64.bin
```

Comment:
I installed `intel-mpi` with Spack and the MPI wrappers definitely weren't in `prefix.bin`. This is the directory they were in for me.